src/storage/v2/inmemory/replication/recovery.cpp

```diff
@@ -117,6 +117,8 @@ std::optional<std::vector<RecoveryStep>> GetRecoverySteps(uint64_t replica_commi
       spdlog::error("Tried to lock a non-existent snapshot path while obtaining recovery steps.");
       return false;
     }
+    spdlog::trace("Found snapshot at path {} with timestamp {}.", latest_snapshot->path,
```

Comment:
maybe delete these logs if you don't think they are necessary, they could cause further logs growth